db/schema.rb

```diff
@@ -10,63 +10,50 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_02_15_165718) do
```

Comment:
- [ ] Hi Currently your migrations are not working and it's because of an error in the schema file. kindly put back the active record line or else your database won't migrate and the database won't work. Also, I would suggest not editing this file manually.